requirements.txt

```diff
@@ -35,6 +36,9 @@ PyGithub
 pytest==2.9.2
 python-bugzilla>=1.2.0
 python-dateutil
+python-keystoneclient
+python-novaclient<7.0
+python-heatclient
```

Comment:
I don't think this should be here since I was removing it because of these requirements being fulfilled by the mgmtsystem package. Didn't you do a wrong rebase?